app/controllers/books_controller.rb

```diff
@@ -0,0 +1,12 @@
+class BooksController < ApplicationController
+  include Wor::Paginate
+  include DeviseTokenAuth::Concerns::SetUserByToken
```

Comment:
Remember what we talked about moving this two includes to the ApplicationController